kafka-ballerina/build.gradle

```
@@ -181,6 +181,7 @@ def disableGroups = ""
 def debugParams = ""
 def balJavaDebugParam = ""
 def testParams = ""
+def testCoverageParam = "--code-coverage --includes=org.ballerinalang.messaging.kafka*:ballerina*"
```

I think this way it is clearer.

```suggestion
def testCoverageParam = "--code-coverage --includes=org.ballerinalang.messaging.kafka.*:ballerina.*"
```